.flake8

```diff
@@ -1,3 +1,3 @@
 [flake8]
-extend-ignore = E265, F401, F541
+extend-ignore = E203, E265, F401, F541
```

Comment:
This is weird.

Reply:
It disallows some valid patterns apparently: https://black.readthedocs.io/en/stable/faq.html#why-are-flake8-s-e203-and-w503-violated